packages/performance/src/performance.ts

```diff
@@ -0,0 +1,274 @@
+import {InflightNavigation} from './inflight';
+import {Navigation} from './navigation';
+import {
+  now,
+  withEntriesOfType,
+  withNavigation,
+  withTiming,
+  supportsPerformanceObserver,
+  referenceTime,
+  hasGlobal,
+} from './utilities';
+import {
+  Event,
+  EventType,
+  LifecycleEvent,
+  TimeToFirstPaintEvent,
+  TimeToFirstContentfulPaintEvent,
+} from './types';
+
+const WATCH_RESOURCE_TYPES = ['script', 'css'];
```

Comment:
Talked IRL. We're going to leave this as-is for now: links are not really blocking/ typically not involved in the context of a navigation, and we will manually instrument TG stuff to avoid lots of noise from various other XHR-using things.